components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -441,6 +441,9 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
         }
 
         const email = await Gapi.getSignedInEmail() || '';
+        this.setState({
+            ['email']: email,
```

Comment:
Kuming, you could write this whole blurb as:

```suggestion
            this.setState({email})
```

Reply:
Reference: https://alligator.io/js/object-property-shorthand-es6/